src/components/gas/GasSpeedButton.js

```diff
@@ -321,19 +322,32 @@ const GasSpeedButton = ({
   }, [currentNetwork, speeds]);
 
   const menuConfig = useMemo(() => {
-    const menuOptions = speedOptions.map(gasOption => ({
-      actionKey: gasOption,
-      actionTitle: upperFirst(gasOption),
-      icon: {
-        iconType: 'ASSET',
-        iconValue: GAS_ICONS[gasOption],
-      },
-    }));
+    const menuOptions = speedOptions.map(gasOption => {
+      const wei =
+        weiToGwei(gasFeeParamsBySpeed[gasOption]?.maxFeePerGas?.amount) +
+        weiToGwei(
+          gasFeeParamsBySpeed[gasOption]?.maxPriorityFeePerGas?.amount,
```

Comment:
i believe you could use `.gwei` instead of `.amount`

Reply:
i tried running it locally and is only taking the `maxFeePerGas` value

Reply:
changed and fixed